Workbooks/Azure Backup/ASR Job History/ASR Job History.workbook

```diff
@@ -0,0 +1,1247 @@
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "value": [
+              "value::all"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "includeAll": false,
+              "selectAllValue": ""
+            },
+            "label": "Workspace Subscription"
+          },
+          {
+            "id": "2373a24f-ad32-4909-a7f6-59b373dcde6c",
+            "version": "KqlParameterItem/1.0",
+            "name": "Workspaces",
+            "label": "Workspace",
+            "type": 5,
+            "description": "LA Workspaces configured in vault diagnostic settings",
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "where type =~ 'microsoft.operationalinsights/workspaces' | project id",
+            "crossComponentResources": [
+              "{Subscriptions}"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources",
+            "value": []
+          },
+          {
+            "id": "254167f1-3459-455a-9bc9-9df92ed8044b",
+            "version": "KqlParameterItem/1.0",
+            "name": "ReplicationScenario",
+            "label": "Replication Scenario",
+            "type": 2,
+            "description": "Filter for replication scenario",
+            "isRequired": true,
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "jsonData": "[    \r\n{ \"value\": \"Azure\", \t\t\t\t\t\t\"label\": \"Azure\" },\r\n{ \"value\": \"Hybrid\", \t\t\t\"label\": \"Hybrid\" }\r\n]",
+            "value": "Azure"
+          }
+        ],
+        "style": "above",
+        "doNotRunWhenHidden": true,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "customWidth": "100",
+      "name": "Filters1"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "_____________________"
+      },
+      "name": "Line1"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "## ASR job history"
+      },
+      "conditionalVisibility": {
+        "parameterName": "Workspaces",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "Heading2"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "This report helps you view information on your successful and failed ASR jobs over a specified duration of time. Currently only jobs triggered on replicated items and recovery plans are shown in this report. [Learn more](https://go.microsoft.com/fwlink/?linkid=2211463&clcid=0x409)"
```

Comment:
Please change to - This report provides information on the successful and failed site recovery jobs over a specified duration of time. Currently only jobs triggered on replicated items and recovery plans are shown in this report.